src/Directory.Build.props

```diff
@@ -23,4 +23,8 @@
     <UnityPath Condition="$(UnityPath) == '' AND Exists('C:\Program Files\Unity\Editor\Data\Managed\UnityEngine.dll')">C:\Program Files\Unity\Editor\Data\Managed</UnityPath>
     <UnityPath Condition="$(UnityPath) == '' AND Exists('/Applications/Unity/Unity.app/Contents/Managed/UnityEngine.dll')">/Applications/Unity/Unity.app/Contents/Managed</UnityPath>
   </PropertyGroup>
+  <PropertyGroup>
+    <UnityTestPath Condition="Exists('C:\Projects\Unity\sentry-unity\samples\unity-of-bugs\Library\ScriptAssemblies\UnityEngine.TestRunner.dll')">C:\Projects\Unity\sentry-unity\samples\unity-of-bugs\Library\ScriptAssemblies</UnityTestPath>

```

Comment:
This one can go?

Reply:
It works for me, if the question about that. Sorry, didn't get it.

Reply:
this assumes `C:\Projects` though? can we use a relative path instead

Reply:
Oh, I see. I'll check.